### ISSUE TITLE
"ddev minio" redirects to "Unresponsive/broken ddev back-end site"

### ISSUE THREAD

**Issue by douggreen** [oblakstudio/ddev-minio#13]
I cannot connect to the minio site. "ddev minio" brings up the URL in my browser but says "Unresponsive/broken ddev back-end site". Has something changed recently (in ddev or in ddev-minio)?

I am using a Drupal 7 site but I've tested this on an OOTB Drupal 10.1 site and get the same error. And just to isolate it further, I created an empty directory 

```
ddev config && ddev get oblakstudio/ddev-minio && ddev minio
```

redirects to a browser that says:

```
502: Unresponsive/broken ddev back-end site.

This is the ddev-router container: The back-end webserver at the URL you specified is not responding. You may want to use "ddev restart" to restart the site.
```

**Comment by rfay** [comment]
With `ddev config --auto` in an empty directory named `junk` I'm able to `ddev minio` and land at the login page at `https://junk.ddev.site:8090/login`

**Comment by douggreen** [comment]
Same problem
```
$ mkdir junk; cd junk; ddev config --auto && ddev get oblakstudio/ddev-minio && ddev minio
```

**Comment by rfay** [comment]
I tested on OrbStack and also on Colima and it worked perfectly.

**Comment by rfay** [comment]
This can be closed. @douggreen reported in Discord that he rebuilt his Colima instance and it was fixed.

**Comment by seebeen** [comment]
I did a complete reimplementation with the latest minio version. Improving the test suite and pushing asap

**Comment by seebeen** [comment]
Fixed in [v1.3.0](/oblakstudio/ddev-minio/releases/tag/v1.3.0)
